### PR TITLE
feat(babel): support babel 6.x.x

### PR DESCRIPTION
**Pull request by neoziro**
- upgrade modules
- upgrade .babelrc
- fix tests
- fix import / export
- fix build script

### CODE REVIEW

**Comment by salper** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,5 @@
 sudo: false
 language: node_js
 node_js:
-- '0.10'
 - '0.12'
-- 'io.js'
+- 'stable'
```
Comment (iamstarkov) [inline .travis.yml]: LTS supported versions are 0.10, 0.12, v4 plus latest one v5

**Comment by nathancahill** [comment]
:+1:

**Comment by neoziro** [comment]
You can use this git branch (https://github.com/neoziro/isparta/tree/babel-6-support-with-lib) as npm package if you want to use isparta with babel 6.

@iamstarkov I changed it because I think it's important to test the latest stable, I think you or @douglasduteil can modify the .travis.yml after merging this PR.

**Comment by scriptjs** [comment]
Please merge this

**Comment by Fluidbyte** [comment]
bump

**Comment by nkbt** [comment]
@neoziro just a little correction:

```sh
npm install neoziro/isparta#babel-6-support-with-lib
```

That worked pretty well for me.

File: .travis.yml (hunk 1 above)
Comment (nkbt) [inline .travis.yml]: Yeah, I would not drop 0.10 for now

**Comment by douglasduteil** [comment]
Alright let's go

**Comment by nkbt** [comment]
woohoo \o/

**Pull request closed without merging**